squidb/src/com/yahoo/squidb/data/adapter/DefaultOpenHelperWrapper.java

```diff
@@ -53,7 +54,11 @@ public void onConfigure(SQLiteDatabase db) {
 
     @Override
     public void onOpen(SQLiteDatabase db) {
-        delegate.onOpen(new SQLiteDatabaseAdapter(db));
+        SQLiteDatabaseAdapter adapter = new SQLiteDatabaseAdapter(db);
+        delegate.onOpen(adapter);
```

Comment:
Well, onConfigure comes before EVERYTHING in AOSP, so this will be a different ordering no matter what, but I suppose we could at least move it before onOpen, sure.